x-pack/test/accessibility/apps/spaces.ts

```diff
@@ -0,0 +1,243 @@
+      await PageObjects.common.navigateToApp('home');
+    });
+
+    it('navigate to manage spaces home page', async () => {
+      await PageObjects.spaceSelector.openSpacesNav();
+      await retry.waitFor(
+        'Manage spaces option visible',
+        async () => await testSubjects.exists('manageSpaces')
+      );
+      await PageObjects.spaceSelector.clickManageSpaces();
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on create space button', async () => {
+      await PageObjects.spaceSelector.clickCreateSpace();
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on input space name', async () => {
+      await PageObjects.spaceSelector.clickEnterSpaceName();
+      await a11y.testAppSnapshot();
+    });
+
+    it('add in space name as space a', async () => {
+      await PageObjects.spaceSelector.addSpaceName('space_a');
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on space avatar to open customize space avatar', async () => {
+      await toasts.dismissAllToasts(); // must clear toasts because they are not accessible
+      await PageObjects.spaceSelector.clickSpaceAcustomAvatar();
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on space initials input in ', async () => {
+      await PageObjects.spaceSelector.clickSpaceInitials();
+      await a11y.testAppSnapshot();
+    });
+
+    it('add in space initials as sa', async () => {
+      await PageObjects.spaceSelector.addSpaceInitials('sa');
+      await a11y.testAppSnapshot();
+    });
+
+    // EUI issue - https://github.com/elastic/eui/issues/3999
+    it.skip('click on the color picker', async () => {
+      await PageObjects.spaceSelector.clickColorPicker();
+      await a11y.testAppSnapshot();
+    });
+
+    // remove the first line once EUI color picker is fixed
+    it('click on a color in color picker in customize space', async () => {
+      await PageObjects.spaceSelector.clickColorPicker();
+      await PageObjects.spaceSelector.setColorinPicker('#2E5C57');
+      await a11y.testAppSnapshot();
+      await browser.pressKeys(browser.keys.ESCAPE);
+    });
+
+    // close custom dialog - adding file upload test here
+    it('click escape to close the custom avatar dialog box', async () => {
+      await browser.pressKeys(browser.keys.ESCAPE);
+      await a11y.testAppSnapshot();
+      await browser.pressKeys(browser.keys.ESCAPE);
+    });
+
+    it('click customize URL identifier', async () => {
+      await PageObjects.spaceSelector.clickOnCustomizeURL();
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on space URL display box', async () => {
+      await PageObjects.spaceSelector.clickOnSpaceURLDisplay();
+      await a11y.testAppSnapshot();
+    });
+
+    it('set the space URL value', async () => {
+      await PageObjects.spaceSelector.setSpaceURL('yay');
+      await a11y.testAppSnapshot();
+    });
```

Comment:
Another case where it doesn't seem to me like checking a11y.testAppSnapshot would find something different before and after you set a URL in an input field.

We should probably just discuss.